apps/passport-server/test/generic-issuance/pipelines/pretix/setupTestPretixPipeline.ts

```diff
@@ -19,7 +19,9 @@ import { expectLength, expectToExist } from "../../../util/util";
 /**
  * Sets up test data required to test {@link PretixPipeline}
  */
-export function setupTestPretixPipeline(): PretixPipelineTestData {
+export function setupTestPretixPipeline(
+  enablePODTickets: boolean = false
```

Comment:
this pipeline can be set up with or without support for v4 identities/pod tickets for testing purposes

Reply:
will this / how will this affect existing pipelines in prod?

Reply:
the way I made it work this PR should basically be a no-op on existing pipelines.